engine/crates/runtime/src/trusted_documents.rs

```diff
@@ -0,0 +1,33 @@
+#[derive(Debug)]
+pub enum TrustedDocumentsError {
+    RetrievalError(Box<dyn std::error::Error + Send + Sync + 'static>),
+    DocumentNotFound,
+}
+
+pub type TrustedDocumentsResult<T> = Result<T, TrustedDocumentsError>;
+
+/// A handle to trusted documents configuration and retrieval.
+pub struct TrustedDocuments {
```

Comment:
It feels like there are too many different things called `TrustedDocuments`. Rename this to `TrustedDocumentsRepository`? It does configuration too, so repository isn't the best. TrustedDocumentsClient?

Reply:
`Client` it is. Pushed.